packages/language-server/test/plugins/typescript/features/FindComponentReferencesProvider.test.ts

```diff
@@ -8,16 +8,16 @@ import { LSAndTSDocResolver } from '../../../../src/plugins/typescript/LSAndTSDo
 import { pathToUrl } from '../../../../src/utils';
 import { serviceWarmup } from '../test-utils';
 
-const testDir = path.join(__dirname, '..');
+const testDir = path.join(__dirname, '..', 'testfiles');
```

Comment:
This warms up the wrong directory. I fixed it while debugging why the test is flaky again.